src/sif/autocost.cc

```diff
@@ -367,16 +369,13 @@ bool AutoCost::Allowed(const baldr::DirectedEdge* edge,
                        const uint64_t current_time,
                        const uint32_t tz_index,
                        int& restriction_idx) const {
-
-  if (flow_mask_ & kCurrentFlowMask) {
-    if (tile->IsClosedDueToTraffic(edgeid))
-      return false;
-  }
+  if (IsClosedDueToTraffic(edgeid, tile))
```

Comment:
same for this one, it should be moved into the `if` below and added as the last condition. this should also be done for all costings that care about traffic not just auto